README.md

```diff
@@ -234,6 +235,25 @@ def validate(sc:SparkContext, config: Contig): SparkJobValidation = {
 }
 ```
 
+### HTTPS / SSL Configuration
+To activate ssl communication, set these flags in your application.conf file (Section 'spray.can.server'):
+```
+  ssl-encryption = on
+  keystore = "~/sjs.jks"
```

Comment:
Does the `~/` actually work?   It's usually better to stick with absolute paths or at least use environment vars, I think this syntax works `${HOME}"/sjs.ks"`